src/broadcast.js

```diff
@@ -0,0 +1,30 @@
+// send new score to API
+
+export const broadcast = async (raw) => {
+  const myHeaders = new Headers();
+  myHeaders.append('Content-Type', 'application/json');
+  const requestOptions = {
+    method: 'POST',
+    headers: myHeaders,
+    body: raw,
+    redirect: 'follow',
+  };
+  const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/dV4aWISG7S2Gi5e0alOq/scores';
+  const request = new Request(requestURL);
+  await fetch(request, requestOptions);
+};
+
+// get scores from API
+
+export const populate = async () => {
+  const requestOptions = {
+    method: 'GET',
+    redirect: 'follow',
+  };
+  const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/dV4aWISG7S2Gi5e0alOq/scores';
+  const request = new Request(requestURL);
+  const response = await fetch(request, requestOptions);
+  const superHeroesText = await response.text();
+  const superHeroes = JSON.parse(superHeroesText);
+  return superHeroes;
```

Comment:
[OPTIONAL]

- [ ] Great effort working on the changes requested by the previous reviewer, But i notice something in your project functional that the previous reviewer could not mention, probably it was an over-sight.
One of the goal of this milestone is the `GET` the score of any user posted via the form whenever the `Refresh` button is clicked on. But currently, in your project the score is automatically added to the scoreboard immediately the user submit a score. which makes the `Refresh` functionality un-achieved. Kindly work on this as this is part of the milestone and it's a very crucial one.
> N.B: I marked this optional because it wasn't raised by the previous review but its a mandatory task for this milestone. Thanks.

Reply:
Hello @brainconnect93 The refresh button is there because if someone else not you have submitted his score you can refresh and see the updated list, and it's not ideal to use the refresh to display the list, otherwise it would be named display not refresh, I hope I have made my idea clear, Thanks for reviewing and also for marking suggestions as optional.